docs/user/OAuth2OIDC-VMware-cloud-services.md

```diff
@@ -0,0 +1,55 @@
+# VMware Cloud Services
+
```

Comment:
I'd add sth like: "this document explains how con configure XXX as an OIDC provider, check out the [parent document](./using-an-OIDC-provider.md) if you are looking for more general information"